docs/community/integrations/fleet_libraries_context.md

```diff
@@ -0,0 +1,194 @@
+# Fleet Context Embeddings - Building a Hybrid Search Engine for the Llamaindex Library
```

Comment:
you prob need to merge main

Reply:
try to fix that pr's lint issue.
that pr breaks the main branch

Reply:
@logan-markewich is this good